lib/puppetserver/ca/utils/http_client.rb

```diff
@@ -175,7 +175,7 @@ def make_store(bundle, crl_usage, crls = nil)
         # Queries the simple status endpoint for the status of the CA service.
         # Returns true if it receives back a response of "running", and false if
         # no connection can be made, or a different response is received.
-        def self.check_server_online(settings, logger)
+        def self.check_server_online(settings, logger, suppress_error = nil)
```

Comment:
Can we default this to `false`, since we expect it to be a boolean?

Reply:
yeah, definitely!